src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -45,6 +47,31 @@ def __init__(self, client):
         self.run_submitter = ExperimentRunSubmitter(self.run_operations)
         self.command_submitter = ExperimentCommandSubmitter(self.run_operations)
 
+    @classmethod
+    def _set_experiment(cls, experiment):
+        """Set experiment info to operation context.
+
+        :param experiment: Experiment name or experiment template absolute path.
+        :type experiment: str
+        """
+        operation_context = OperationContext.get_instance()
```

Comment:
add TODO that we should pass through the operation_context when invoke flow or script node cross process.

Reply:
removed operation context